### ISSUE TITLE
Custom struct tags

### ISSUE THREAD

**Issue by adrianlungu** [mgechev/revive#1508]
#### **Is your feature request related to a problem? Please describe.**
Go validator allows defining custom struct tags used to validate various input. Revive currently works only with prebuilt struct tags; this means any custom struct tag will be flagged by the linter. 

#### **Describe the solution you'd like**
Config defined struct tags to allow a project to define custom tags which the linter should consider ok. 

#### **Describe alternatives you've considered**
Disabling the struct tag linter !?

#### **Additional context**

If we have a i.e. `maxdecimal` struct tag, the linter will yield

```
struct-tag: unknown option "maxdecimals" in validate tag (revive)
```

**Comment by chavacava** [comment]
Hi @adrianlungu, thank you for reporting the issue.

Did you try to configure the rule to accept the option?
```yaml
[rule.struct-tag]
arguments = ["validate,maxdecimals"]
```

**Comment by adrianlungu** [comment]
Oh, saw the rule but did not see that it allows user defined configuration, that's perfect, thank you @chavacava !

**Comment by chavacava** [comment]
> Oh, saw the rule but did not see that it allows user defined configuration, that's perfect, thank you [@chavacava](https://github.com/chavacava) !

[Here](https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#struct-tag) a short doc on the conf of the rule.
